plugin.video.filmarkivet/default.py

```diff
@@ -0,0 +1,123 @@
+# -*- coding: utf-8 -*-
+
+'''
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+'''
+
+import os, sys
+import xbmc, xbmcaddon, xbmcgui, xbmcplugin, xbmcvfs
+from lib.filmarkivet import Filmarkivet
+import requests
+
+try:
+    # Python 3
+    from urllib.parse import parse_qs
+except ImportError:
+    # Python 2
+    from urlparse import parse_qs
+
+addon = xbmcaddon.Addon()
+ADDON_PATH = addon.getAddonInfo('path')
+__translation = addon.getLocalizedString
+
+
+def view_menu(menu):
+    for item in menu:
+        title = u'{} | {}'.format(item.title, item.description) if item.description else item.title
+        li = xbmcgui.ListItem(label=title)
```

Comment:
You'll also have a performance increase if you create your plugin items with `offscreen=True`. See docs: https://codedocs.xyz/xbmc/xbmc/group__python__xbmcgui__listitem.html#ga15bcb930a4f91dd97f8ed7c633a20618